app/src/main/java/de/westnordost/streetcomplete/util/CrashReportExceptionHandler.kt

```diff
@@ -109,4 +118,9 @@ class CrashReportExceptionHandler(
     private fun deleteCrashReport() {
         appCtx.deleteFile(crashReportFile)
     }
+
+    private fun readLogFromDatabase(): String = logsController
+        .getLogs(LogsFilters())
+        .take(ApplicationConstants.MAX_LOG_LINES_TO_ATTACH_TO_CRASH_REPORT)
```

Comment:
As noted before, I think it makes more sense to limit them by time rather than total number. Or at least, *additionally* by time.